tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -1166,6 +1166,18 @@ static bool passCursorInfoForDecl(
       Symbols.pop_back();
     }
   }
+  for (auto D : Info.ShorthandShadowedDecls) {
+    CursorSymbolInfo &SymbolInfo = Symbols.emplace_back();
+    DeclInfo DInfo(D, Type(), /*IsRef=*/true, /*IsDynamic=*/false,
+                   ArrayRef<NominalTypeDecl *>(), Invoc);
+    if (auto Err = fillSymbolInfo(SymbolInfo, DInfo, MainModule, Info.Loc,
+                                  AddSymbolGraph, Lang, Invoc, PreviousSnaps,
+                                  Allocator)) {
+      // Ignore but make sure to remove the partially-filled symbol
+      llvm::handleAllErrors(std::move(Err), [&](const llvm::StringError &E) {});
```

Comment:
```suggestion
      llvm::handleAllErrors(std::move(Err), [](const llvm::StringError &E) {});
```
Doesn't really matter 😅